nrf_wifi/fw_if/umac_if/inc/fw/host_rpu_sys_if.h

```diff
@@ -56,6 +56,10 @@
 
 #define NRF_WIFI_COUNTRY_CODE_LEN 2
 
+#ifndef BIT_POS
+#define BIT_POS(nr) (1 << nr)
```

Comment:
I guess you did this as a WAR for Linux compilation? But this isn't right, we can solve it properly in many ways, simplest would be to handle specifically for Linux:

```
#ifdef __ZEPHYR__
#ifndef BIT
#define BIT(nr)  (1 << nr)
#endif
#endif

Reply:
Well, Zephyr also has the `BIT` macro, see https://github.com/nrfconnect/sdk-zephyr/blob/main/include/zephyr/sys/util_macro.h#L36, so, just a matter of including proper headers

Reply:
Using __ZEPHYR__ will introduce OS specific handling in OSAL code.

Also using this difn't help for Linux as BIT() is not defined for linux. 
Adding BIT() for linux also result s in redifinition as it's defined as 
#define BIT(nr)                 (UL(1) << (nr)).

Is there ohter way?

Reply:
> Using ZEPHYR will introduce OS specific handling in OSAL code.

Yes, but we handle it for all OSes.

> Also using this difn't help for Linux as BIT() is not defined for linux.

It it defined for both Zephyr and Linux, you need to include proper header files.